src/helpers/getResultStringForArrayWithoutItemRef.ts

```diff
@@ -0,0 +1,34 @@
+import { ResultStringForArrayWithoutItemRef, SwaggerProps } from '../types';
+import { parseFormat } from './parseFormat';
+import { parseProperty } from './parseProperty';
+import { parseRefType } from './parseRefType';
+
+export const getResultStringForArrayWithoutItemRef = ({
```

Comment:
We didn't have an helper for this before, instead it was found here https://github.com/LandrAudio/openapi-codegen-typescript/pull/53/files#diff-2df477f1aaf80cfbe0f1dfdd0f04f882bc895bd3bcec81e55658851864eaa519L205